src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -121,11 +121,11 @@ public ProAI(final String name, final String type) {
 
   public static void Initialize(final TripleAFrame frame) {
     LogUI.initialize(frame); // Must be done first
-    LogUtils.log(Level.FINE, "Initialized Hard AI");
+    ProLogUtils.log(Level.FINE, "Initialized Hard AI");
```

Comment:
Violation of DRY having "Level.FINE" repeated many times. I recommend creating another method signature for log in ProLogUtils that takes just a string and defaults to Level.FINE.